Update RoCur guide for curators

reikookamoto opened the pull request:
Summary of edits:

- Remove any mention of Twitter/TweetDeck
- Change "tweet" to "post" (a platform-agnostic verb)
- Replaced/removed broken links (e.g., some blog posts were no longer available)
- Revised phrasing to improve parallelism and reorganized content to cluster related concepts together for better readability and logical flow

To-do:
- Add instructions on how to use app passwords in the FAQ
- Add instructions on how to work with Alt Text in the FAQ

reikookamoto commented:
@sndaba I moved the Curator Procedures section from #158 to the How Curating Works section of the curator guide.

sndaba reviewed (approved):
@reikookamoto Update on adding links, wording and moved the Pre-Curation check list section from the end of the page to make it part of How curation works section to make to combine curation operation.

reikookamoto commented:
> @reikookamoto In the curators guide, I updated and added some links where applicable, adjusted wording and moved the Pre-Curation check list section from the end of the page to make it part of _How curation works_ section to combine curation operation.
> 
> In the admin page, I added appropriate links and put in some further explanations in the responsibilities section. I made sure the curation ends on Saturday instead of Sunday in all the pages even on the airtable forms. I also made the changes in the airtable schedule that shows the curation ends on a Saturday.
> 
> **Things to do:** Please remember to add the _when someone is nominated_ template in the admin page from the google drive folder.
> 
> Hope it helps.

Yup, will work on the "When someone is nominated" template and push again when it's done.

reikookamoto commented:
@sndaba Addressed the remaining items on the to-do list; I had to write an email template from scratch because I can no longer access the old gmail account, but I think I covered everything that needed to be included!

sndaba commented:
> @sndaba Addressed the remaining items on the to-do list; I had to write an email template from scratch because I can no longer access the old gmail account, but I think I covered everything that needed to be included!
> 
> Also edited some links since links to different sections within the same document don't need a full URL in Markdown. Also, using an in-document link typically prevents the web browser from reloading the page or opening it in a new window.

@reikookamoto Ok, cool. Thanks for changing the linking to sections within the document, its more polished.
I thought I was the only one who couldn't access the gmail account. 
@shannonpileggi 
How will we access the google drive that has the template for the curator's image? If we can't access it, we will have to create one from scratch as well and edit the pages that mention google drive.

reikookamoto commented:
@shannonpileggi merge ready
